test/html.js

```diff
@@ -21,6 +21,16 @@ describe('#html', function () {
     expect(result).to.equal('It is <span class="wmlime">green</span>')
   })
 
+  it('auto-closes open tags at end of multi-line string', function () {
+    const result = html('It is {lime}green\n on a new line \r another \r\n last one.')
+    expect(result).to.equal('It is <span class="wmlime">green\n on a new line \r another \r\n last one.</span>')
+  })
+
+  it('replaces colors in multi-line string', function () {
+    const result = html('It is {lime}green\n on a new line \r another \r\n last one.{/}')
```

Comment:
Thoughts on changing this so the closing tag is not actually at the end of the string (perhaps in the middle of the second line)? So that we can test the closing tag is being inserted where we want it in a multiline scenario, and ensuring it's not being caused by auto-closure.

Reply:
no problem, so it's like this now if that's what you mean. Works fine still

```
 const result = html('It is {lime}green\n on a new line{/} \r another \r\n last one.')
 expect(result).to.equal('It is <span class="wmlime">green\n on a new line</span> \r another \r\n last one.')
```